hbase-balancer/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -486,26 +485,12 @@ public static void scanMeta(Connection connection, @Nullable final byte[] startR
 
     if (LOG.isTraceEnabled()) {
       LOG.trace("Scanning META" + " starting at row=" + Bytes.toStringBinary(startRow) +
-        " stopping at row=" + Bytes.toStringBinary(stopRow) + " for max=" + rowUpperLimit +
+        " stopping at row=" + Bytes.toStringBinary(stopRow) + " for max=" + maxRows +
```

Comment:
This may log a negative number?

Reply:
I think it is fine? We know that -1 means no limit here.